File: nelio-alves/algoritmo-e-logica-de-programacao/java/src/exercicios/NegativosMatriz.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class NegativosMatriz {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int M, N;

	    System.out.print("Qual a quantidade de linhas da matriz? ");
		M = sc.nextInt();
	    System.out.print("Qual a quantidade de colunas da matriz? ");
		N = sc.nextInt();

	    int[][] mat = new int[M][N];

		for (int i = 0; i < M; i++) {
			for (int j = 0; j < N; j++) {
	        	System.out.printf("Elemento [%d,%d]: ", i, j);
				mat[i][j] = sc.nextInt();
	        }
	    }
		
	    System.out.println("VALORES NEGATIVOS:");
		for (int i = 0; i < M; i++) {
			for (int j = 0; j < N; j++) {
				if (mat[i][j] < 0) {
	            	System.out.printf("%d\n", mat[i][j]);
	            }
	        }
	    }

		sc.close();
	}
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/negativos_matriz.py
M: int; N: int

M = int(input("Qual a quantidade de linhas da matriz? "))
N = int(input("Qual a quantidade de colunas da matriz? "))

mat: [[int]] = [[0 for x in range(N)] for x in range(M)]

for i in range(M):
	for j in range(N):
		mat[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("VALORES NEGATIVOS:")
for i in range(M):
	for j in range(N):
		if mat[i][j] < 0:
			print(mat[i][j])